src/views/Home.vue

```diff
@@ -58,8 +58,10 @@ export default {
     RuleCard,
   },
   methods: {
-    optionPage() {
-      router.push({ name: 'newRule' });
+    newRule() {
+      const rule = this.$store.getters.rule(this.id);
```

Comment:
`this.id` n'est pas défini. On peut enlever le paramètre, et le rendre optionnel dans le getter ?